### ISSUE TITLE
Installed APK not find TTF asset anymore

### ISSUE THREAD

**Issue by Alisis33** [radimitrov/CSharpShellApp#170]
V2.5.123
A TTF asset is used correcfly in chshel, but not in installed APK.

**Comment by radimitrov** [comment]
True... This worked when I tested it last week as far as I recall.

**Comment by Alisis33** [comment]
Could be /\ related?
This is the only difference I found in chproj.

**Comment by radimitrov** [comment]
/\ related? Can you elaborate? Uppercase? Assembly resources themselves are case sensitive, but that's not the issue.

**Comment by Alisis33** [comment]
I compared a old and a new csproj file, the only difference relsted to TTF was the path changed the /\. 2020+ this should be irrelevant for a OS, but...

I made a smal test project. The TTF shows up correctly under cshel, but not in a installed APK.

Maybe check the path in witch it searches the TTF, and if it can find it.

It stop working after 2.5.123.

**Comment by radimitrov** [comment]
Might be that. For paths Android/Unix only uses "/" and not "\" like Windows. And looks like I don't have .Replace("\\", "/") there to auto fix that.

**Comment by Alisis33** [comment]
Some when bevore V2.5.123 it worked, but now even the xamarin example not work anymore.
What ever you fix/update then, probable break it.
Not mater how I use the ttf, the apk not find it. I think it is how you access the assets, or a support/permission is not in the apk version. Sadly there is no error message or any hint.

(I will update as soon I am sure to not loose the pruchase, and test it again.)

https://github.com/xamarin/xamarin-forms-samples/tree/main/WorkingWithFonts

**Comment by Alisis33** [comment]
I update it to 6.12.0
It work now.
